### ISSUE TITLE
(WCAG 2.1) Accordion text styling for static text questions is not accessible

### ISSUE THREAD

**Issue by jeanniefu** [civiform/civiform#3560]
Required to meet WCAG 2.1 guidelines. Identified during an accessibility audit against WCAG 2.1.

**WCAG SC**:  [2.1.1. Keyboard](https://www.w3.org/TR/WCAG21/#keyboard)

**Observed results**: For static text questions, we support some light formatting including an accordion style format See "What are eligibility requirements?" in screenshot). This needs some additional fixes to be accessible:
- It is not focusable or cannot be activated by keyboard only.
- Does not convey role (e.g. button) and state (expanded/collapsed)

**How to fix**:
- Ensure all interactive elements receive keyboard focus or can be activated with the keyboard. Use `<button>` element or if using ARIA, see https://www.w3.org/WAI/ARIA/apg/example-index/disclosure/disclosure-faq.html for example
- Ensure custom controls provide proper textual name, role, and state information. See https://www.w3.org/WAI/ARIA/apg/example-index/disclosure/disclosure-faq.html for example

![image](https://user-images.githubusercontent.com/9094240/192905876-3fe76b63-5de4-483f-8602-eadacbb0bd5c.png)

**Comment by clouser2** [comment]
Something that may be useful for implementation: https://accessibility.huit.harvard.edu/technique-expandable-sections
